Source/src/core/serialization/TypeConverter.h

```diff
@@ -7,6 +7,7 @@
 #include <Math/Quat.h>
 #include <Math/float4x4.h>
 #include <assimp/vector3.h>
+#include <assimp/matrix4x4.h>
```

Comment:
This gave problems when including this file on Gameplay project, as there were templates defined with this type. This file however worked before on Hachiko as probably this file is either included before including this whole header, or hepch.h already has this. Adding this here was a simpler solution to not crowd the gameplaypch with internal assimp classes.